pages/index.html

```diff
@@ -4,9 +4,29 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>✨Kassidy's Portfolio Site✨</title>
+  <link rel="stylesheet" href="../styles/style.css">
 </head>
-<body>
-  
-</body>
+<header>
+  <h1>K. Buslach</h1>
+  <h1>ADA C17 Sharks 🦈</h1>
+
+      <p>Welcome to my portfolio!
+        Here is what you'll find here:
+    <ul>
+		<nav>
+      <li><a href="../index.html">Home</a></li>
+			<li><a href="/about.html">About me</a></li>
+			<li><a href="/portfolio.html">Portfolio</a></li>
+		</nav>
+    </ul>
+      </p>  
+</header>
+<footer>
+  <p class="first">Connect with me on Linkedin! </p>
+ 
+	<a class="middle"href="https://www.linkedin.com/in/kassidy-buslach-3a8b4921b"><img class="LI" src="../images/linkedin.png"
+		alt="Linkedin icon to navigate to Kassidy's Linkedin"></a>
+		<p class="last">Copyright 2022</p>
+</footer>
```

Comment:
You can wrap from line 10 where you header starts to here where your footer ends with a body element